### ISSUE TITLE
explore ancestry.com connections (was: track down a handful (two or three) addresses for the sample page)

### ISSUE THREAD

**Issue by nicholasjhorton** [STAT210-S23/Parsons_Paper_Register#5]
these can be accessed through `ancestry.com` through the Amherst College library, city directories from the appropriate era

**Comment by nicholasjhorton** [comment]
@nicholasjhorton's suggestion: let's create a new folder in this repo called `addresses` which contains the images from ancestry.com as well as the single jpg file that you used.

**Comment by KYEYOUNGLEE** [comment]
https://docs.google.com/spreadsheets/d/1NS2lsPjFTYz8EwyM5D4OeYa73nDQdQ_XWdGeGko3G6g/edit?usp=sharing

**Comment by KYEYOUNGLEE** [comment]
- I used the sample page we used in class!
- just wrote down birthplaces in the google sheet
- address in comment below

==> takeaways - a lot of people come from Ireland. People's family names reappear when looking at spouse/parents

**Comment by KYEYOUNGLEE** [comment]
<<<address>>>
<William Connor>
Holyoke Ward 3, Hampden, Massachusetts, USA
Holyoke, Hampden, Massachusetts, USA
5 Avon Place, Holyoke, MA
96 Beech St., Holyoke
Bridge Street, Holyoke, Hampden, Massachusetts, USA
Avon Place, Holyoke Ward 3, Hampden, Massachusetts, USA

<Martin Kennedy>
Holyoke Ward 1, Hampden, Massachusetts, USA
Holyoke, Hampden, Massachusetts, USA
Holyoke, Hampden, Massachusetts, USA
Holyoke, Massachusetts, USA
Northampton Ward 2, Hampshire, Massachusetts, USA
Holyoke, Massachusetts, USA

<Frank L Russell>
West Springfield, Hampden, Massachusetts, USA
Springfield Ward 7, Hampden, Massachusetts, USA
West Springfield, Hampden, Massachusetts, USA
Bridge, West Springfield, Hampden, Massachusetts, USA
West Springfield, West Springfield, Hampden, Massachusetts, USA
Continental Street, Springfield Ward 7, Hampden, Massachusetts, USA

**Comment by nicholasjhorton** [comment]
This is really cool: thanks for the update and your work on this front.

**Comment by nicholasjhorton** [comment]
I've incorporated the prior work tracking down info from Ancestry.com. See https://github.com/STAT210-S23/Parsons_Paper_Register/blob/main/addresses_exploration/ancestry_addresses.csv

We will need someone to pick up this project.

**Comment by nicholasjhorton** [comment]
@LorraineOloo and @Cheafitz25 I look forward to your next steps on this front. 

My sense is that you'll want to:

1. merge the two spreadsheets and create a new spreadsheet that can be used for later analysis
2. write up a bit of background about how the ancestry information was collected
3. summarize what we know about one or two of the individuals

You will want to create a Quarto file (and formatted pdf) in https://github.com/STAT210-S23/Parsons_Paper_Register/tree/main/addresses_exploration

Is it feasible to be able to move forward with this and then share your results

**Comment by Cheafitz25** [comment]
Hello @nicholasjhorton, Lorraine and I are working on finding records on ancestry.com but they want us to pay to use the database. Do you have any ideas to work around this?

**Comment by nicholasjhorton** [comment]
Sorry for the complication.

Can you please confirm that you are logged in via the Amherst College library account for ancestry? That should allow you full access.

**Comment by Cheafitz25** [comment]
I think we are figuring it out right now. Should be fine now thank you.

**Comment by LorraineOloo** [comment]
Here is the link to our work/results: https://github.com/STAT210-S23/Parsons_Paper_Register/blob/d57e183b244a02dccd8f12d75dfcd92004367e9c/addresses_exploration/Parsons'%20Paper%20Company%20Workers%20Database.qmd

**Comment by Cheafitz25** [comment]
Professor, we are having trouble making csv file viewable on the qmd. Can you help us at some point, please?

**Comment by nicholasjhorton** [comment]
I've renamed the file (see https://github.com/STAT210-S23/Parsons_Paper_Register/commit/a67f84ac1bda83d432fcf5de9bc1d07b1e4fbd32).

Which spreadsheet are you trying to access? If it's the version we saved on Google Drive, you'll need to set up authentication with Google.

Or is it the initial spreadsheet I created? That can be found here: https://github.com/STAT210-S23/Parsons_Paper_Register/blob/main/sample_pages/Parsons_page253.csv

Or is it the spreadsheet with new info that I added to this repo? https://github.com/STAT210-S23/Parsons_Paper_Register/blob/main/addresses_exploration/ancestry_addresses.csv

Both these files can be read directly into R using relative pathnames.

**Comment by nicholasjhorton** [comment]
Once you are all set with merging the two spreadsheets, I wonder whether it would be helpful to start to create some screenshots to illustrate your methods and results. This is something that we might be able to share with Eileen in advance of our meeting with her next week.

I suspect that using the existing Quarto file to create this technical report would be workable. Thoughts?

**Comment by Cheafitz25** [comment]
Hello Professor. Our problem was making the new spreadsheet that we created viewable on the qmd file @LorraineOloo attached above. Here it is again; https://github.com/STAT210-S23/Parsons_Paper_Register/blob/d57e183b244a02dccd8f12d75dfcd92004367e9c/addresses_exploration/Parsons'%20Paper%20Company%20Workers%20Database.qmd

**Comment by Cheafitz25** [comment]
I think it is a simple code error which I am unable to work through.

**Comment by nicholasjhorton** [comment]
See https://github.com/STAT210-S23/Parsons_Paper_Register/blob/main/addresses_exploration/workers_database.pdf for a first cut reading the files that I think that you need to read and merge. Am I missing something?

**Comment by nicholasjhorton** [comment]
Note that I had to remove the header line from the https://github.com/STAT210-S23/Parsons_Paper_Register/blob/main/addresses_exploration/ancestry_addresses.csv file.

**Comment by Cheafitz25** [comment]
I think I am misinterpreting the problem. Lorraine and I have already created a spreadsheet compiled of the workers that we were able to find the most information on.  We linked it in the file we created yesterday but I don't think you can click on the link and see the spreadsheet. How can I fix the code to make the link take me to the spreadsheet?

**Comment by nicholasjhorton** [comment]
Now I'm really confused.

How did you create the spreadsheet that you are trying to read? Can you download and add this spreadsheet to the repo? That would be the best approach I think. (Does the spreadsheet replace what Kyeyoung had created?)

I tried to access the link that you provided but it wasn't shared with me.

**Comment by Cheafitz25** [comment]
We created our own spreadsheet from scratch on Google Sheets and then added the entries from prior lists. https://docs.google.com/spreadsheets/d/1urqUmn_mIRrHS15O5r5G6eSOLpbY2G2-X1pkkrzG3Fo/edit#gid=0 
This is the link to it.

**Comment by nicholasjhorton** [comment]
Sounds good: thanks for letting me know.

You have two choices:

1. You can set up Google Authentication and use the googlesheets4 package to access the spreadsheet directly.
2. You can make it world accessible and then the Authentication is much simpler.
3. A third option (for now) would be to download the spreadsheet and add the spreadsheet to the repo.

In any case, can you please add a Quarto file that describes your methods for how you tracked down this information? I'm thinking that this would be a technical appendix that would allow others to do the same linkage for other employees in the register.

**Comment by LorraineOloo** [comment]
Hi Nick, could you please help us insert a proper link to a website and photos in the qmd file?
We have pushed the needed files in the class repo already so that you can see them.

**Comment by nicholasjhorton** [comment]
Here you go: https://github.com/STAT210-S23/Parsons_Paper_Register/commit/db21fb063d7aeec1ee7fce1ff3337c68bf757525

It's good to see how things are moving forward.

During today's class it would be helpful to split this issue up into multiple issues for next steps for these analyses (this issue is getting hard to follow).

**Comment by LorraineOloo** [comment]
Make sense of the addresses.  
Create an appendix section.

**Comment by nicholasjhorton** [comment]
I look forward to reviewing your draft prior to class. Sometime today can you please:

- push changes to the formatted pdf, and
- add a new comment to let me know you are ready for me to read your revised report?

**Comment by nicholasjhorton** [comment]
This looks great: see some minor edits at https://github.com/STAT210-S23/Parsons_Paper_Register/commit/e85eecd15a92b93ab8cc1992aed57a9225c7b78e

I'm closing this issue and will send this off to Eileen.
